cst/plugins/eu.esdihumboldt.cst.functions.collector/plugin.xml

```diff
@@ -15,7 +15,6 @@
       <propertyFunction
             category="eu.esdihumboldt.hale.align.general"
             cellExplanation="eu.esdihumboldt.cst.functions.collector.explanations.AssignFromCollectorExplanation"
-            icon="icons/augmentation.gif"
```

Comment:
Is there another icon available that would fit? Or should we look for one/create one specifically for this function?